joomla/joomla-cms
# [3.10] Improve pre update checker cs and add a dedicated message when no extensions are found

### Summary of Changes

[3.10] Improve pre update checker cs and add a dedicated message when no extensions are found

### Testing Instructions

- Install a plain 3.10-dev without extensions
- add this update server: https://update.joomla.org/core/nightlies/next_major_list.xml
- notice the message that no extensions have been found.

### Expected result

A message that no extensions have been found.

### Actual result

Only the notices part of the table with no indication what is happening or what this is reffering to.

### Documentation Changes Required

none
